src/wallet/wallet.h

```diff
@@ -741,6 +745,106 @@ class CAccountingEntry
     std::vector<char> _ssExtra;
 };
 
+enum NoteType {
+    sapling,
+    sprout
+};
+
+class NotesHelperObject
```

Comment:
```suggestion
class NotesIndexEntry
```
or something like that. (Avoid vague name components like "helper" or "object" that don't add useful information about what the thing is.)

Reply:
https://github.com/zcash/zcash/pull/4399/commits/5d118077d75ad09decac01ba241d29486b9dad17